### PR TITLE
Allow `SK_STRESS_FILE_FILTER` to take a comma-separated list of files to stress test

### PR DESCRIPTION
**Pull request by ahoppen**
This way, we can ask the stress tester to test two files without needing to recompile everything twice.

### CODE REVIEW

**Comment by ahoppen** [comment]
@swift-ci Please test

File: SourceKitStressTester/Sources/SwiftCWrapper/SwiftCWrapper.swift [hunk 1]
```diff
@@ -88,8 +88,10 @@ public struct SwiftCWrapper {
 
     // Determine the list of stress testing operations to perform
     let operations = swiftFiles.flatMap { (file, sizeInBytes) -> [StressTestOperation] in
-      if let fileFilter = ProcessInfo.processInfo.environment["SK_STRESS_FILE_FILTER"], !file.contains(fileFilter) {
-        return []
+      if let fileFilter = ProcessInfo.processInfo.environment["SK_STRESS_FILE_FILTER"] {
+        if !fileFilter.split(separator: ",").contains(where: { file.contains($0) }) {
+          return []
+        }
```
Comment (bnbarham) [inline SourceKitStressTester/Sources/SwiftCWrapper/SwiftCWrapper.swift:94]: Could do the split outside of the flatMap so it's not done over and over, but doesn't *really* matter here.

**Review by bnbarham — APPROVED** [review]

File: SourceKitStressTester/Sources/SwiftCWrapper/SwiftCWrapper.swift (hunk 1 above)
Comment (ahoppen) [reply SourceKitStressTester/Sources/SwiftCWrapper/SwiftCWrapper.swift:94]: I think splitting a string by `,` is the least of our performance concerns for the stress tester and I’d prefer to keep the entire filtering logic localized to a few lines.

Comment (bnbarham) [reply SourceKitStressTester/Sources/SwiftCWrapper/SwiftCWrapper.swift:94]: It'd only be one more line, not really any less localized :P. But as I said, it doesn't matter... was just something I noticed at first glance.

Comment (ahoppen) [reply SourceKitStressTester/Sources/SwiftCWrapper/SwiftCWrapper.swift:94]: I still prefer it like this 😉

**Pull request merged**